galaxy/values.yaml

```diff
@@ -75,6 +75,41 @@ tolerations: []
 
 affinity: {}
 
+# extraInitContainers:
+#   - name: my-first-container
+#     applyToJob: true
+#     applyToWeb: true
+#     image: "{{.Values.image.repository}}:{{.Values.image.tag}}"
+#     command: ['sh', '-c', 'cp "/galaxy/server/config/job_conf.xml.sample_advanced" {{.Values.persistence.mountPath}}/']
+#     volumeMounts:
+#       - name: galaxy-data
+#         mountPath: "{{.Values.persistence.mountPath}}"
+#   - name: my-second-container
+#     applyToJob: true
+#     applyToWeb: false
+#     image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+#     command: ['sh', '-c', 'cp "/galaxy/server/config/galaxy.yml" {{.Values.persistence.mountPath}}/']
+#     volumeMounts:
+#       - name: galaxy-data
+#         mountPath: "{{.Values.persistence.mountPath}}"
+
+# initScripts:
```

Comment:
This is `initScripts` instead of `extraInitScripts` but it's `extraInitContainers` instead of `initContainers`? Should they be the same?

Reply:
My thought was that `extraInitContainers` indicate that there are already `initContainers` running by default and do disambiguate between the k8s native `initContainers` key and the one in `values`, so I think it almost has to be `extraInitContainers` imo. On the other hand, we are not injecting any `initScripts` by default right now, and many other charts use `initScripts`: https://github.com/helm/charts/search?q=initScripts&unscoped_q=initScripts vs https://github.com/helm/charts/search?q=extrainitscripts&unscoped_q=extrainitscripts vs https://github.com/search?q=extrainitscripts&type=Code . However, it does seem less parallel esthetically so I don't mind switching the `initScripts` to `extraInitScripts` if we don't care to follow other charts that much. Someone has to do it first ultimately haha.